test/unit/views/CookiesSpec.scala

```diff
@@ -87,329 +87,203 @@ class CookiesSpec
-        "'This helps us identify how you use GOV.UK so we can make the site better', '2 years'" in new Fixture {
-          val expectedTableAnswersText: List[String] =
-            List("_ga", "This helps us identify how you use GOV.UK so we can make the site better", "2 years")
-
-          view.verifyTableRowText(tableId = "cookies-analytics-table", expectedTableAnswersText, rowNumber = 3)
-        }
-      "have the following answers in the table 'ga_nextpage_params', " +
-        "'This stores data to be sent to Google on the next page you request', 'When you close your browser'" in new Fixture {
-          val expectedTableAnswersText: List[String] = List(
-            "ga_nextpage_params",
-            "This stores data to be sent to Google on the next page you request",
-            "When you close your browser"
-          )
-
-          view.verifyTableRowText(tableId = "cookies-analytics-table", expectedTableAnswersText, rowNumber = 4)
-        }
-      "have a pragraph of 'You can opt out of Google Analytics cookies (opens in new tab).'" in new Fixture {
-        view
-          .select("#cookies-analytics-opt-out")
-          .text mustBe "You can opt out of Google Analytics cookies (opens in new tab)."
-      }
-      "have a link of 'https://tools.google.com/dlpage/gaoptout'" in new Fixture {
-        view.select("a[href*=\"tools.google\"]").first.attr("href") mustBe "https://tools.google.com/dlpage/gaoptout"
+
+      "have a paragraph of descriptive text" in new Fixture {
+        view.select("#cookies-analytics-paragraph-1").size() mustBe 1
       }
-    }
-    "Comparing different versions" should {
-      "have a heading of 'Comparing different versions of a webpage (Optimizely)'" in new Fixture {
-        view.select("#cookies-versions-heading").text mustBe "Comparing different versions of a webpage (Optimizely)"
+
+      "have a table with the following headings 'Name', 'Purpose', 'Expires'" in new Fixture {
+        view.verifyTableHeadings("cookies-analytics-table", expectedTableHeadingsText)
       }
 
-      "have a paragraph of 'Optimizely software is used to test different versions of a webpage (or webpages) " +
-        "to see which performs better. This is done to help make improvements and ensure the site is " +
-        "meeting the needs of its users.'" in new Fixture {
+      "have the expected cookies listed in the table" in new Fixture {
+        val expectedCookies: List[String] = List(
+          "_gat",
+          "GDS_successEvents and GDS_analyticsTokens",
+          "_ga",
+          "ga_nextpage_params"
+        )
 
-          val paragraph: String = "Optimizely software is used to test different versions of a webpage (or webpages) " +
-            "to see which performs better. This is done to help make improvements and ensure the site is meeting the needs of its users."
+        view.verifyTableContainsCookieName("cookies-analytics-table", expectedCookies)
```

Comment:
agree we shouldn't be unit testing the content here, but might we want to test the `Expires` values?